torch/future_div.py

```diff
@@ -0,0 +1,6 @@
+import torch
+
+torch.div = torch.true_divide
```

Comment:
there's some existing thing like torch.future or torch.__future__ already, let's put it there.

Reply:
This is moot. We removed the future div stuff.